#!/usr/bin/env python
PACKAGE = "arduino_imu_9dof"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()
gen.add("yaw_calibration", double_t, 0, "Yaw Calibration", 0, -10, 10)

exit(gen.generate(PACKAGE, "arduino_imu_9dof", "imu"))
